packages/react-components/react-theme/package.json

```
@@ -30,7 +29,8 @@
     "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
-    "tslib": "^2.1.0"
+    "tslib": "^2.1.0",
+    "@fluentui/tokens": "^1.0.0-alpha"
```

should use a pinned version for pre-releases
```suggestion
    "@fluentui/tokens": "1.0.0-alpha"
```